### PR TITLE
Paul homework 5 6

### PR DESCRIPTION
**Pull request by pvanwert4**

### CODE REVIEW

File: Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb [hunk 1]
```diff
@@ -26,10 +26,8 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": 2,
```
Comment (shubags) [inline Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb:29]: This is the dumbest implementation I've ever seen.

File: Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb [hunk 2]
```diff
@@ -75,7 +71,7 @@
        "  </thead>\n",
        "  <tbody>\n",
        "    <tr>\n",
-       "      <th>0</th>\n",
+       "      <td>0</td>\n",
```
Comment (shubags) [inline Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb:74]: I love this.

File: Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb [hunk 3]
```diff
@@ -187,21 +184,26 @@
...
-   "outputs": [],
-   "source": []
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Female: Economics\n",
```
Comment (shubags) [inline Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb]: Can you please change her major to Physics?

Comment (pvanwert4) [reply Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb]: No, requires registrar approval

**Comment by shubags** [comment]
Looks great! A++++++++++

You're not failing anymore.

**Pull request merged**